tests/appsec/test_ip_blocking_maxed.py

```diff
@@ -0,0 +1,115 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+import json
+
+from tests.remote_config.test_remote_configuration import rc_check_request
+from utils import weblog, context, coverage, interfaces, released, rfc, bug, irrelevant, scenarios
+from utils.tools import logger
+
+with open("tests/appsec/rc_expected_requests_block_ip_maxed_asm_data.json", encoding="utf-8") as f:
+    EXPECTED_REQUESTS = json.load(f)
+
+
+@rfc("https://docs.google.com/document/d/1GUd8p7HBp9gP0a6PZmDY26dpGrS1Ztef9OYdbK3Vq3M/edit")
+@released(cpp="?", dotnet="2.16.0", php_appsec="0.7.0", python="?", ruby="?", nodejs="3.11", golang="1.47.0")
+@released(
+    java={
+        "spring-boot": "0.110.0",
+        "sprint-boot-jetty": "0.111.0",
+        "spring-boot-undertow": "0.111.0",
+        "spring-boot-openliberty": "0.115.0",
+        "ratpack": "1.7.0",
+        "jersey-grizzly2": "1.7.0",
+        "resteasy-netty3": "1.7.0",
+        "vertx3": "1.7.0",
+        "*": "?",
+    }
+)
+@irrelevant(
```

Comment:
Given this line in the scenario `weblog_env={"DD_APPSEC_RULES": None},` I think the `@irrelevant` annotation is not needed. It won't hurt either.